src/azure-cli-core/azure/cli/core/decorators.py

```diff
@@ -95,8 +95,10 @@ def _wrapped_func(*args, **kwargs):
     return _decorator
 
 
-def transfer_doc(source_func):
+def transfer_doc(source_func, *additional_source_funcs):
```

Comment:
The additional use case is a rare use case. Concatenating docstring is not use case in any other command module anytime soon. Therefore it is not ideal to modify the decorator just to fit a corner case. Please revoke the change and write a customized decorator in your own module.